webui/react/src/hooks/useModal/UserSettings/useModalCreateGroup.tsx

```diff
@@ -125,14 +127,18 @@ const ModalForm: React.FC<Props> = ({ form, users, group, groupRoles }) => {
               optionFilterProp="children"
               placeholder={'Add Roles'}
               showSearch>
-              {knownRoles.map((r) => (
-                <Select.Option
-                  disabled={groupRoles?.find((gr) => gr.id === r.id)?.fromWorkspace?.length}
-                  key={r.id}
-                  value={r.id}>
-                  {r.name}
-                </Select.Option>
-              ))}
+              {Loadable.match(knowRolesLoadable, {
+                Loaded: () =>
+                  knownRoles.map((r) => (
+                    <Select.Option
+                      disabled={groupRoles?.find((gr) => gr.id === r.id)?.fromWorkspace?.length}
+                      key={r.id}
+                      value={r.id}>
+                      {r.name}
+                    </Select.Option>
+                  )),
+                NotLoaded: () => undefined,
```

Comment:
https://github.com/determined-ai/determined/pull/5522#discussion_r1042317154

We should add loading state (spinner) or `// TODO show spinner when this is loading`